### DISCUSSION TITLE
Gameplay for osu!catch lazer feels completely different and is more difficult than stable

### DISCUSSION THREAD

**Discussion by Faustas156** [ppy/osu#29297] [Ideas]
Hi, for context, I am currently top #1,500 on osu!catch with 6.6k pp. I have not played for pp in about a year or two and thought that lazer ctb would be a great way to come back to the game and derust. Unfortunately, I have come across a slew of issues as I continued to play thru the gamemode on lazer. 

The main issues I have encountered all happen to be with gameplay. Most of them just happen to be with the general "feel" of catch on lazer, the gameplay feels incredibly slippery and feels incredibly off, I feel like i'm going a bit too fast with my catcher or it feels incredibly inconsistent during certain points. This ultimately results in me being incredibly uncomfortable in playing the gamemode on lazer. I was barely able to FC 3-4 stars when I was playing cause of weird misses that I know should not have happened, and unfortunately it really didn't not get better with time either. I spent a solid week trying to adapt, but as soon as I felt like I was getting slightly better, the gameplay started to feel weird again and I felt like I was going a step down everytime I tried to boot up the game to improve.

After returning to osu!stable to play a multiplayer game with a friend, I had noticed that the gameplay felt so much more tighter and more comfortable with stable than lazer. It's a little bit indescribable but everything felt "right", it just felt like as if I had returned to the game as if I was playing it as usual and I was able to land a lot of the stuff I wasn't really able to on lazer. I was almost able to match some of my top PP plays with maps ranging around 5.3 - 5.8 stars, whilst on lazer, I was failing most of those maps, not even 20 seconds in.

There may also be a possible issue with hyperdashing, it feels like as if they slightly push you too far or too close in certain instances, making the transition from one part of the map to the other much harder than it should be. It is also quite indescribable but I feel very "off" with a lot of maps that include that. 

The difference between osu!stable gameplay and osu!lazer gameplay is honestly night and day, osu!stable feels more comfortable and more consistent than osu!lazer. I really wish that the gameplay could be better improve for osu!lazer as it makes playing catch incredibly hard on it.

These are all the issues I've noticed so far and if there are any other note-worthy issues, I'll be sure to update this. Sorry for the incredibly long text, but I was quite frustrated with this and really wanted to get this out here.

**Comment by peppy** [comment]
As far as I know we've done frame comparisons and they match perfectly. Maybe you can setup input to broadcast to both games and do a comparison if you're convinced otherwise?

**Reply by Faustas156** [reply]
> Maybe you can setup input to broadcast to both games and do a comparison if you're convinced otherwise?

I would like to, but i'm not sure on which tools I could use to best replicate that, I don't have a second monitor that would let me run both games in a comfortable way
